### PR TITLE
GitHub buttons

### PR DESCRIPTION
**Pull request by lankaukk**
Added colors to the themes for both light and dark mode. 

Created the `<RoundButton>` component to display the number of remote changes and local changes in the title bar. 

Added appropriate colors and styling to these buttons
GitHub pane:
• Pull Remote Changes
• Push Local Changes 
• Authenticate With GitHub 
Title Bar:
• # Remote Changes
• # Local Changes
• Connect to GitHub
• Sign In To Save

Bonus: Fixed the project title to the _dead center_ of the title bar, by adding flex properties to the left and right siblings.

Screen Shot 2022-11-20 at 11 11 39 PM

### CODE REVIEW

File: editor/src/components/titlebar/buttons.tsx [hunk 1]
```diff
@@ -9,56 +13,60 @@ interface TitleButtonProps {
...
   onClick,
   color,
   children,
 }) => {
   const bgColor = color == null ? {} : { backgroundColor: color }
   const ButtonEl = styled(TitleButton)((props) => ({
-    padding: '3px 12px 3px 8px',
-    height: 36,
+    padding: '3px 12px',
+    //margin: '0px 3px',
```
Comment (ruggi) [inline editor/src/components/titlebar/buttons.tsx:53]:
```suggestion
```

File: editor/src/components/titlebar/title-bar.tsx [hunk 2]
```diff
@@ -79,42 +107,63 @@ const TitleBar = React.memo(() => {
...
       <div>
         <ProjectTitle>{projectName}</ProjectTitle>
       </div>
-
       <div style={{ display: 'flex', height: '100%' }}>
         <div style={{ display: 'flex', alignItems: 'center', paddingRight: 24 }}>
           <TestMenu />
         </div>
-        {/* <TextButton onClick={exportToGithub}>Fork</TextButton> */}
+        {/* <SquareButton onClick={}>Fork</SquareButton> */}
```
Comment (ruggi) [inline editor/src/components/titlebar/title-bar.tsx:162]:
```suggestion
```

**Review by ruggi — APPROVED** [review]
LGTM - I made a couple of quick fixes here https://github.com/concrete-utopia/utopia/pull/2882/commits/d0a76d7b2c14557c2814f1789fedfd2478aeabbc due to timezone 😴

**Review by balazsbajorics — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]

**Comment by ruggi** [comment]
gonna merge this as it introduces merge conflicts on the branch flow one I'm working on, so I can fix them

**Pull request merged**